pages/index.js

```diff
@@ -1,15 +1,58 @@
+import Header from "@/components/Header";
+import TransactionForm from "@/components/TransactionForm";
 import TransactionsList from "@/components/TransactionsList";
+import { transactions } from "@/lib/transactions";
+import { useState } from "react";
+import styled from "styled-components";
+import { ulid } from "ulid";
+import useLocalStorageState from "use-local-storage-state";
 
 export default function HomePage() {
+  const [transactionsList, setTransactionsList] = useLocalStorageState(
+    "transactions",
+    { defaultValue: transactions }
+  );
+  const [successMessage, setSuccessMessage] = useState("");
+
+  function handleAddTransaction(data) {
+    setTransactionsList([
+      { ...data, id: ulid(), currency: "EUR" },
+      ...transactionsList,
+    ]);
+    setSuccessMessage("Transaction successfully added!");
+    setTimeout(() => {
+      setSuccessMessage("");
+    }, 3000);
+  }
+
   return (
     <>
-      <header>
-        <h1>Money Manager</h1>
-      </header>
+      <Header />
       <main>
         <h2>Transactions</h2>
-        <TransactionsList />
+        <TransactionForm onAdd={handleAddTransaction} />
+        {successMessage && (
+          <StyleSuccessMessage>{successMessage}</StyleSuccessMessage>
+        )}
+        <TransactionsList transactions={transactionsList} />
       </main>
     </>
   );
 }
+
+const StyleSuccessMessage = styled.p`
+  position: fixed;
+  bottom: 20px;
+  left: 50%;
+  transform: translateX(-50%);
+  color: var(--text-color-dark);
+  font-size: 12px;
```

Comment:
If you use `rem` as font-size unit, it is more accessible.

Reply:
Yes, we have changed it.